python/remote_api/src/lynx/kite.py

```diff
@@ -884,6 +884,40 @@ def f(**kwargs):
     _add_documentation_to_operation(f, name)
     return f
 
+  @property
+  def vertices(self) -> 'SingleOutputAtomicBox':
+    '''Returns the vertices as a table.'''
+    return self._select_all('vertices')
+
+  @property
+  def edges(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edges as a table.'''
+    return self._select_all('edges')
+
+  @property
+  def graph_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the graph attributes as a table.'''
+    return self._select_all('graph_attributes')
+
+  @property
+  def edge_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edge attributes as a table.'''
+    return self._select_all('edge_attributes')
+
+  @property
+  def segmentations(self):
+    '''Access a given segmentation.
```

Comment:
Maybe mention that it uses a _Use segmentation as base graph_ box. That would clarify what kind of access it gives. (E.g. no belongs-to edges.)

Reply:
Given that I changed the `graph.segmentations` to a function call that now contains a one-liner that shows the `takeSegmentationAsBaseGraph` method, I think this is not really required anymore?

Reply:
> Given that I changed the `graph.segmentations` to a function call that now contains a one-liner that shows the `takeSegmentationAsBaseGraph` method, I think this is not really required anymore?

The documentation shouldn't assume that the user has read the source code. 😄